src/Behat/Behat/Hook/Call/RuntimeFeatureHook.php

```diff
@@ -40,10 +42,19 @@ public function __construct($scopeName, $filterString, $callable, $description =
         parent::__construct($scopeName, $filterString, $callable, $description);
 
         if ($this->isAnInstanceMethod()) {
+            if (is_array($callable)) {
+                $className = $callable[0];
+                $methodName = $callable[1];
+            } else {
+                $reflection = new ReflectionMethod($callable);
+                $className = $reflection->getDeclaringClass()->getShortName();
```

Comment:
why the shortname (which is not unique) ?

Reply:
This is only used to print out an error message. Using the short class name makes the error message clearer and easier to read and I think that the case of having duplicate context class names and method names would be so rare that I don't think we should worry about using the full name

Reply:
Agreed, I think short name is preferable here